src/applications/mhv-secure-messaging/components/ComposeForm/CantFindYourTeam.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { VaAdditionalInfo } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { mhvUrl } from '~/platform/site-wide/mhv/utilities';
+import { isAuthenticatedWithSSOe } from '~/platform/user/authentication/selectors';
+
+const CantFindYourTeam = () => {
+  const fullState = useSelector(state => state);
+  const [prefLink, setPrefLink] = useState('');
+
+  useEffect(
+    () => {
+      setPrefLink(mhvUrl(isAuthenticatedWithSSOe(fullState), 'preferences'));
+    },
+    [fullState],
+  );
```

Comment:
I think `useMemo` would be more appropriate and efficient here
```suggestion
  const prefLink = useMemo(
    () => mhvUrl(isAuthenticatedWithSSOe(fullState), 'preferences'),
    [fullState],
  );
```

Reply:
this is solid 🚀